pkg/artifactory/resource_artifactory_local_generic_repository.go

```diff
@@ -15,5 +15,6 @@ func resourceArtifactoryLocalGenericRepository(pkt string) *schema.Resource {
 		repo := unpackBaseRepo("local", data, pkt)
 		return repo, repo.Id(), nil
 	}
-	return mkResourceSchema(baseLocalRepoSchema, inSchema(baseRemoteSchema), unpack, constructor)
+	baseLocalRepoSchema := getBaseLocalRepoSchema(pkt)
+	return mkResourceSchema(baseLocalRepoSchema, inSchema(baseLocalRepoSchema), unpack, constructor)
```

Comment:
👍🏼

Reply:
probable existing issue fixed. Previously `inSchema(baseRemoteSchema)` has been changed to `inSchema(baseLocalRepoSchema)`. Using BaseLocalRepoSchema instead of BaseRemoteRepoSchema.